crates/re_viewer/src/blueprint/.gitattributes

```diff
@@ -1,5 +1,4 @@
 # DO NOT EDIT! This file is generated by crates/re_types_builder/src/lib.rs
 
 .gitattributes linguist-generated=true
-mod.rs linguist-generated=true
-panel_view.rs linguist-generated=true
+validation.rs linguist-generated=true
```

Comment:
seems wrong. `validation.rs` is _not_ a generated file

Reply:
Good catch. I think this was an intermediate state before I changed some paths/names.